README.zh.md

```diff
@@ -1,159 +1,75 @@
-# RROS
+<div align=center>
+<img src=https://github.com/Richardhongyu/RROS/assets/33805041/d315241e-5a35-4a36-b0c4-1cd2255b0e0a width=60% />
+</div>
 
-[![Documentation](https://img.shields.io/badge/view-docs-blue)](https://bupt-os.github.io/website/docs/guides/example-guide/)
-<!-- [![ci tests](https://github.com/rust-real-time-os/RTOS/actions/ci.yml/badge.svg)](TODO: add a new CI in this repo) -->
+--------------------------------------------------------------------------------
+
+[![Documentation](https://img.shields.io/badge/view-docs-blue)](https://BUPT-OS.github.io/website/docs/)
+[![.github/workflows/ci.yaml](https://github.com/BUPT-OS/RROS/actions/workflows/ci.yaml/badge.svg)](https://github.com/BUPT-OS/RROS/actions/workflows/ci.yaml)
 [![Zulip chat](https://img.shields.io/badge/chat-on%20zulip-brightgreen)](https://rros.zulipchat.com/)
 ![RROS](https://img.shields.io/badge/RROS-0.0.1-orange)
-[![zh](https://img.shields.io/badge/lang-zh-yellow.svg)](https://github.com/BUPT-OS/RROS/blob/master/README.zh.md)
 [![en](https://img.shields.io/badge/lang-en-yellow.svg)](https://github.com/BUPT-OS/RROS/blob/master/README.md)
+[![zh](https://img.shields.io/badge/lang-中文-yellow.svg)](https://github.com/BUPT-OS/RROS/blob/master/README.zh.md)
 
 RROS（Rust实时操作系统）是一个双内核操作系统，由实时内核（使用Rust编写）和通用内核（Linux）组成。 RROS几乎可以兼容所有的Linux程序，并提供比RT Linux更好的实时性能。RROS目前正在作为在轨卫星载荷的操作系统进行实验[“天算星座”项目](http://www.tiansuan.org.cn/)。
 
+可以从[这里](https://bupt-os.github.io/website/docs/introduction/架构图.png)看到RROS的架构图和一个[演示视频](https://bupt-os.github.io/website/docs/introduction/demo.mp4)。
+
+## 新闻
 
-你可以在这里找到我们的[架构图](https://bupt-os.github.io/website/architecture.png)
+- [2023.11.30] RROS 在 Xenomai meetup 2023 上展示（[照片](https://bupt-os.github.io/website/news/2023_11_30/xenomai_workshop/)）。

```

Comment:
The Chinese and English versions here are inconsistent, and the link of news `RROS is successfully launched into space! ` needs to be modified.

Reply:
Fixed.